src/osd/osd_types.h

```diff
@@ -1517,10 +1517,11 @@ struct pg_pool_t {
     return peering_crush_bucket_count != 0;
   }
 
-  bool stretch_set_can_peer(const set<int>& want, const OSDMap& osdmap,
+  bool stretch_set_can_peer(const set<int>& want, const OSDMap *osdmap,
```

Comment:
Why this change?  It appears that osdmap is assumed to be non-null, so normally a ref would be the order of the day?

Reply:
There was something in a dropped commit that made it easier this way, but that's not true any more and you're right. Changed!